great_expectations/datasource/data_connector/configured_asset_dbfs_data_connector.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from typing import Optional
+
+import great_expectations.exceptions as ge_exceptions
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetFilesystemDataConnector,
+)
+from great_expectations.datasource.data_connector.asset import Asset
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
+
+
+class ConfiguredAssetDBFSDataConnector(ConfiguredAssetFilesystemDataConnector):
+    """
+    Extension of ConfiguredAssetFilesystemDataConnector used to connect to the DataBricks File System (DBFS)
+
+    DataConnectors produce identifying information, called "batch_spec" that ExecutionEngines
+    can use to get individual batches of data. They add flexibility in how to obtain data
+    such as with time-based partitioning, splitting and sampling, or other techniques appropriate
+    for obtaining batches of data.
+
+    The ConfiguredAssetDBFSDataConnector is one of two classes (InferredAssetDBFSDataConnector being the
+    other one) designed for connecting to data on DBFS.
+
+    A ConfiguredAssetDBFSDataConnector requires an explicit specification of each DataAsset you want to connect to.
+    This allows more fine-tuning, but also requires more setup.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        datasource_name: str,
+        base_directory: str,
+        assets: dict,
+        execution_engine: Optional[ExecutionEngine] = None,
```

Comment:
Should execution engine be optional (or allow None?) I think Alex recently changed things to make execution_engine required. _get_full_file_path_for_asset requires it

Reply:
seconded. and I believe this is the relevant PR : https://github.com/great-expectations/great_expectations/pull/3643

Reply:
Great catch, resolved in https://github.com/great-expectations/great_expectations/pull/3659/commits/4c0f3b5623d6e94673fd4235823db879e154263f